packages/block-library/src/breadcrumbs/index.php

```diff
@@ -85,7 +85,11 @@ function render_block_core_breadcrumbs( $attributes, $content, $block ) {
 			$breadcrumb_items = array_merge( $breadcrumb_items, block_core_breadcrumbs_get_terms_breadcrumbs( $post_id, $post_type ) );
 		}
 		// Add current post title (not linked).
-		$breadcrumb_items[] = sprintf( '<span aria-current="page">%s</span>', get_the_title( $post ) );
+		$title = get_the_title( $post );
+		if ( empty( $title ) ) {
```

Comment:
`empty()` can have some special cases and isn't the best choice here. For example, `$a = '0'; var_dump( empty( $a ) )` would return `true`. So maybe we can do this instead:

```suggestion
		if ( strlen( $title ) === 0 ) {
```

Also note `strlen()` isn't quite accurate either: if `$title` is a single emoji, it will return some other number like `3` or `4`, but it's enough for this use case IMO. If we want to get accurate, we can use `mb_strlen()` but I don't think it's necessary here.